Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/ItemsPage.xaml

```diff
@@ -25,32 +25,32 @@
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <ScrollView Grid.Row="1">
-            <RefreshView x:DataType="local:ItemsViewModel" Command="{Binding LoadItemsCommand}" IsRefreshing="{Binding IsBusy, Mode=TwoWay}">
```

Comment:
Removing this button does not seem like a good idea as long as there is no alternative for it. (Clicking on an available device should open the signal-strength page which was the whole idea of issue #70 , not add the available device as a paired one)

Reply:
I would disagree and vote for adding two buttons as shown in this draft:
Clicking on the Name should open the page to add it
Clicking on the Bluetooth should switch to the strength page
And Clicking on the Pin should open the map page
![buttons](https://user-images.githubusercontent.com/52490096/171384833-5cdd775e-9e11-4b47-9f68-468e218e825e.jpg)